roles/imagemagick/defaults/main.yml

```diff
@@ -3,7 +3,17 @@
 project_base: /opt
 install_path: "{{ project_base }}/install"
 magick_path: "{{ install_path }}/imagemagick_sources" # needs double-quotes for yaml syntax
+
 openjpg_ver: '2.1.0'
-libtiff_ver: '4.0.5'
-libpng_ver: '1.6.20'
-imagemagick_ver: '.' # call the role with imagemagick_ver: '6.8' to get the most recent 6.8 release, '.' to match the newest release
+openjpg_256: 1232bb814fd88d8ed314c94f0bfebb03de8559583a33abbe8c64ef3fc0a8ff03
+
+libtiff_ver: '4.0.6'
+libtiff_lib_ver: 5.2.4
```

Comment:
I don't think so, the version numbers are unquoted in the ffmpeg PR and it seems to work.